Gemfile

```diff
@@ -80,6 +80,8 @@ group :development, :test do
   gem 'shoulda-matchers', git: 'https://github.com/thoughtbot/shoulda-matchers.git', branch: 'rails-5'
   # Trace routes
   gem 'traceroute'
+  # Database cleaner
+  gem 'database_cleaner'
```

Comment:
Why do you need database cleaner?

Reply:
My tests failed with duplicate (unique constraint failed or sth like that) which means that the factory is creating something that's already in the database.
Working solution: either use database_cleaner, or db:drop and db:setup everytime

Reply:
Oh I think I know that. Yeah sometimes Rails has this stupid thing where if some of the tests fail then the db might not be cleaned properly. But if all the tests succeed, Rails should always properly clean the db. can you try?

Reply:
Hmm since my db has been cleaned by `database_cleaner`, I'm not sure how to reproduce this :sweat_smile:

Reply:
Why not remove the gem and all mentions of the cleaner, then `bin/rails db:drop db:setup` then run `rspec` twice?

Reply:
Alright, will do this for now. I'll keep in mind your suggestion next time I encounter the same error.